src/components/Tooltip/index.js

```diff
@@ -174,7 +160,7 @@ class Tooltip extends PureComponent {
             tooltipTextStyle,
             pointerWrapperStyle,
             pointerStyle,
-        } = getTooltipStyles(
+        } = this.state.isReady ? getTooltipStyles(
```

Comment:
Is this necessary? We are not using any of these styles anyway unless `this.state.isReady === true`.

Reply:
I think it's better to just move this style calculations to `TooltipRenderedOnPageBody` as they are only used there.